.github/compatibility-test-matrices/main/client-chain-a.json

```diff
@@ -18,6 +18,7 @@
     "TestClientTestSuite"
   ],
   "test": [
+    "TestClientUpdateProposal_Succeeds",
```

Comment:
I believe we needed to add [this test](https://github.com/cosmos/ibc-go/pull/4758) back, right?

Reply:
I'm don't think so. It only really needs to be run on the version we are releasing. This is single chain functionality, so there's no reason to expect changes on main to affect an already released version for client recovery. I think it was unnecessary to have it in compatibility test. We should ensure it is run before new releases though

Reply:
I remove from `main` and leave it in the others then?

Reply:
If we want to limit the amount of tests run, then I think we can:
- make a new file `proposal-chain-a.json` which has the proposal types that primarily only need to be tested on one chain, maybe `single-chain-test.json` works as well (could include genesis restart/upgrades)
- set the matrix to test chain-a version as the release branch and chain-b version as the release branch (no need to set many versions for chain-b)

Is this mostly correct, @chatton? Does this make sense @crodriguezvega?

I expect the following tests to be single chain:
- TestClientUpdateProposal_Succeeds 
- TestRecoverClient_Succeeds (>= v8)
- TestClient_Update_Misbehaviour
- TestAllowedClientsParam
- TestScheduleIBCUpgrade_Succeeds
- TestMaxExpectedTimePerBlockParam
- UpgradeTests
- TestIBCGenesis

Reply:
yep we can remove the `chain-b` json file, and keep a single item under the `chain-b` in `proposal-chain-a.json`/`single-chain-test.json` (I prefer `single-chain-test.json` as a name).

Reply:
I updated the PR. I removed the `...-chain-b.json` files where needed, but didn't rename the `...-chain-a.json` files or created a new `single-chain-test.json` file. The reason is that we already had a `localhost-transfer-chain-a.json` file to run the localhost tests and instead of grouping all the single chain tests in the same file, I thought it would still be nice to group them by topic (client, connection, etc). If you don't agree with this, I can still change it.